mysql-test/suite/rocksdb.rpl/t/suite.opt

```diff
@@ -1 +1 @@
-$ROCKSDB_OPT $ROCKSDB_LOAD_ADD
+$ROCKSDB_OPT $ROCKSDB_LOAD_ADD --loose-rocksdb_unsafe_for_binlog=true
```

Comment:
This is a bit complicated. Facebook really only wants to support ROW with MyRocks but AFAIK there is no practical way to enforce this and IMHO it is just not right. MyRocks should be tested as much as possible with all formats. This option disables some of their (incomplete) checks for things that would be unsafe if mixed with other options like read free, etc... that are not validated in combination with this option check. So without this option, most of these newly added tests would only work for ROW.